components/bulletins/BulletinCard.vue

```diff
@@ -50,55 +50,56 @@ export default {
                 bulletinCard: true,
             }
         },
+        showButton() {
+            return false
+        },
     },
 }
 </script>
 
 <style lang="postcss" scoped>
 .bulletinCard {
-    @apply relative flex flex-col items-center font-serif rounded-3xl border-4;
-    width: 47%;
-    @media (min-width: 415px) {
-        width: 214px;
-    }
-    border-color: #4b4b4b;
-    color: #4b4b4b;
+    @apply relative flex flex-col justify-center items-center rounded-xl;
+    min-width: 236px;
+    min-height: 310px;
+    color: #a9a6d9;
+    background: #1f1c3b;
 }
 
 .bulletinCard:hover {
-    border-color: #413f5a;
+    border-color: #746bb8;
+    background: #352d66;
+    color: #f0ebf5;
 }
 
 /* A trick to set height to be equal to width. */
 /* Checkout: https://stackoverflow.com/a/6615994/7969188 */
 .bulletinCard > .dummy {
     margin-top: 230%;
-    @media (min-width: 370px) {
+    @media (min-width: 375px) {
         margin-top: 170%;
     }
 }
 
 .bulletinCard__content {
     @apply absolute;
+    top: 120px;
+    padding: inherit;
 }
 
 .bulletinCard__content > h2 {
     @apply text-base text-center mt-6 mb-5 font-bold;
-    @media (min-width: 415px) {
+    font-family: 'Noto Serif TC';
```

Comment:
**question (non-blocking):** same question as [here](https://github.com/pycontw/pycontw-frontend/pull/227/files#r901009674)

Reply:
Have to replace 'Source Sans Pro' w/ 'Noto Serif TC' according to the spec

Reply:
Thanks for the reminder. I didn't notice that.